site/_docs/java-tools.md

```diff
@@ -11,6 +11,7 @@ supports both the local file system and HDFS.
 
 The subcommands for the tools are:
 
+  * bloom-filter (since ORC 2.1) - check the bloom filter of the specified column
```

Comment:
- Since we can backport `tools` module patch, let's change this to `ORC 2.0.1`.
- The command name, `bloom-filter`, is a little ambiguous. Could you provide more intuitive name?

Reply:
> let's change this to ORC 2.0.1.

OK

> The command name, bloom-filter, is a little ambiguous.

How about using `check-bloom-filter`?

Reply:
Sounds better to me for this context.

BTW, if you have more plan to extend this commend, you can use `check` as subcommand or argument, too.

Reply:
Thank you very much for your suggestion. 
Let us use the `check` command. Based on this, we can add min and max statistics to check later.